### ISSUE TITLE
ToAu 22 

### ISSUE THREAD

**Issue by Vampric** [DarkstarProject/darkstar#5883]
**_I have:_**

- [] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:** 30190328_2

**_Source Branch_** (master/stable) **:** "Steps to reproduce: 
Entering BCNM for Aht Urgan Mission 22, Khimaira 13 is extremely weak. has less than 5,000 hp, npc karaba still does not assist or cast."

**_Additional Information_** (Steps to reproduce/Expected behavior) **:**

**Comment by helixhamin** [comment]
None of the wikis show the HP, so that could be a problem.
Perhaps if someone can get an estimate based on retail, we could adjust, but without having a value, no clue how to fix it. From what I see here though, http://spiritangel.catsboard.com/t892-mission-22-shield-of-diplomacy it looks like it should be close to 30k HP.
The NPC issue will take some ai-ing...

**Comment by Vampric** [comment]
yes should be 20 ~ 30k only way of getting value was from using addon scoreboard last time I fought it before updates I just know it is not suppose to die in 3 hits

**Comment by Vampric** [comment]
https://www.youtube.com/watch?v=6yQD8mweEu0

**Comment by Vampric** [comment]
take 4min and view this video post I seen several 2 to 5k weaponskills adding up roughly 20,000 hp

**Comment by Vampric** [comment]
also it did not die in 3 hits ^^
